test/Microsoft.TestPlatform.AcceptanceTests/AcceptanceTestBase.cs

```diff
@@ -46,17 +38,16 @@ public class AcceptanceTestBase : IntegrationTestBase
     public const string CoreRunnerTargetRuntime = "";
     public const string InIsolation = "/InIsolation";
 
-    public const string NETFX452_48 = "net452;net461;net472;net48";
-    public const string NETFX451_48 = "net452;net461;net472;net48";
+    public const string NETFX462_48 = "net462;net472;net48";
     public const string NETCORE21_50 = "netcoreapp2.1;netcoreapp3.1;net5.0";
-    public const string NETFX452_NET50 = "net452;net461;net472;net48;netcoreapp2.1;netcoreapp3.1;net5.0";
-    public const string NETFX452_NET31 = "net452;net461;net472;net48;netcoreapp2.1;netcoreapp3.1";
-    public const string DEFAULT_RUNNER_NETFX = "net451";
+    public const string NETFX462_NET50 = "net462;net472;net48;netcoreapp2.1;netcoreapp3.1;net5.0";
+    public const string NETFX462_NET31 = "net462;net472;net48;netcoreapp2.1;netcoreapp3.1";
+    public const string DEFAULT_RUNNER_NETFX = "net462";
```

Comment:
```suggestion
    public const string DEFAULT_RUNNER_NETFX = Net462TargetFramework;
```

Reply:
makes it a bit harder to read without much benefit imho. 

I dont want this ```public const string NETFX462_NET50 = "net462;net472;net48;netcoreapp2.1;netcoreapp3.1;net5.0";``` to become a huge string full of constants.